### PR TITLE
Submitted liyuan.ts

### PR DESCRIPTION
**Pull request by lyuanww**

### CODE REVIEW

File: submission-folder/liyuan.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
+/**
+ * HOMEWORK
+ * rename this file to *your name*.ts
+ * Translate this JS code into valid TypeScript that follows good practices,
+ * following instructions in the comments
+ */
+
+// replace all `var` declarations appropriatly
+
+const NINE_THOUSAND : number = 9000
```
Comment (marcus-ong-qy) [inline submission-folder/liyuan.ts:10]: no need to explicitely assign type if it's implied by TypeScript already

File: submission-folder/liyuan.ts [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+
+a === 1 && console.log('a is one')
+
+b.length > 0 && console.log('b is a non-empty string')
+
+c.length === 3 && c[0] === 3 && console.log('c is an array of length 3, and its first element is the number 3')
+
+
+// use ternary operator
+let s : String = 'is a loooooooooong string'
```
Comment (marcus-ong-qy) [inline submission-folder/liyuan.ts:81]: same, no need to explicitly type all variables. Also, always use lowercase `string` for string types

**Review by marcus-ong-qy — CHANGES_REQUESTED** [review]
lgtm! just note that you don't always need to explicitely assign a type ya

File: submission-folder/liyuan.ts (hunk 1 above)
Comment (lyuanww) [reply submission-folder/liyuan.ts]: fixed in c1a1105cebd79ab481debf3da9c66a071f307d76

File: submission-folder/liyuan.ts (hunk 2 above)
Comment (lyuanww) [reply submission-folder/liyuan.ts]: fixed in c1a1105cebd79ab481debf3da9c66a071f307d76

**Review by marcus-ong-qy — APPROVED** [review]
lgtm!

**Pull request merged**